compareECLFiles.cmake

```diff
@@ -449,7 +449,7 @@ if(MPI_FOUND)
                       FILENAME SPE1CASE1
                       SIMULATOR flow
                       ABS_TOL ${abs_tol}
-                      REL_TOL ${rel_tol}
+                      REL_TOL 0.01
```

Comment:
Is this intentional?

Reply:
Yes. It means this test will not (or at least in a very strict sense) detect numerical changes, but the purpose of the test is to detect changes to the HDF5 writing and ensure that the Damaris subsystem works.

Reply:
>  the purpose of the test is to detect changes to the HDF5 writing and ensure that the Damaris subsystem works.

Ah, okay.  In that case this looks good to me.